.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+    - bugfix/**
+    - feature/**
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+**
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Prepare Test Files
+      run: ln -s $(pwd)/src/query_parser/codex ontology
+
+    - name: Build the stack
+      run: docker-compose up -d
+
+    - name: Test Query Endpoint
+      run: docker run --network container:codex-flare_flare_1 -v $(pwd)/src/query_parser/i2b2:/i2b2 appropriate/curl -vX POST --data @/i2b2/i2b2_gecco_demo.xml -H "Content-type:i2b2/xml" -H "Accept:internal/xml" http://localhost:5000/query
```

Comment:
This does not need to be changed per se, but some thoughts on this one:

Testing the project should happen in code. Otherwise we always have to adjust the pipeline in order implement some made up tests. I see where this is coming from since the project itself does not have any tests (at first glance) but then we don't need a test job. I would argue this job should be implemented here as soon as there is a single test that can be ran in an automated fashion.

As it is now, the tests would be incomplete because there is another parser that needs to be tested.

Reply:
....but the thing that we can do here is introduce CodeQL to check python code.